packages/runtime-core/src/apiInject.ts

```diff
@@ -45,7 +44,7 @@ export function inject(
 ) {
   // fallback to `currentRenderingInstance` so that this can be called in
   // a functional component
```

Comment:
Just wondering, is there any point in this comment now?

Reply:
I'm not sure, so I didn't delete it.